src/adapters/lokijs/worker/index.worker.js

```diff
@@ -1,10 +1,22 @@
 // @flow
 /* eslint-disable no-restricted-globals */
+/* eslint-disable global-require */
+/* eslint-disable import/no-mutable-exports */
 
 import LokiWorker from './lokiWorker'
 
-if (process.env.NODE_ENV !== 'test') {
+let defaultExport
+
+// In a web browser, Webpack will spin up a web worker and run this code there, while the importing
+// module will see a Worker class.
+// But Jest will actually import this file and has to provide a Worker interface, so we export a mock
+
+if (process.env.NODE_ENV === 'test') {
+  const workerMock = require('./workerMock').default
+  defaultExport = workerMock
+} else {
   self.workerClass = new LokiWorker(self)
+  defaultExport = self
 }
 
-export default self
+export default defaultExport
```

Comment:
I think we can avoid mutating variable and disabling eslint rule by creating a helper function. Something like this:
```
const getDefaultExport = () => {
  if (process.env.NODE_ENV === 'test') {
    const workerMock = require('./workerMock').default
    return workerMock
  }

  self.workerClass = new LokiWorker(self)
  return self
}
export default getDefaultExport()
```
WDYT?

Reply:
👍